_posts/2016-06-20-foreman-community-newsletter-june-2016.md

```diff
@@ -0,0 +1,136 @@
+appropriate backups before upgrading. Do get in touch on IRC or the mailing
+lists if you have any questions!
+
+[k30]:  https://groups.google.com/d/topic/foreman-users/KlmzfqIcrRQ/discussion
+[k301]: https://groups.google.com/d/topic/foreman-users/vcjo8CW6p9I/discussion
+
+### SSL comes to (yum.|deb.|)theforeman.org
+
+Thanks to the hard work of the infra team, our website and packaging repos now
+have LetsEncrypt SSl certificates in place. This has the main benefit of
+allowing secure downloads of our packaging GPG keys for verification of the
+packages themselves.
+
+The quickstart guides have been [updated][gpg] to reflect the updated URL scheme.
+
+[gpg]: https://theforeman.org/manuals/1.11/quickstart_guide.html#QuickstartGuide
+
+### New media and blogs
+
+#### [Community Demo](https://youtu.be/C_BebxMpW5c) on 19th May 2016
+
+On this episode of the Foreman Community Demo, we cast a wide net to pull in
+updates from Pulp (the project that handles content for Katello), from Katello,
+from our CLI testers, and some Infoblox DNS/DHCP providers for the Smart Proxy.
+There's also a quick look at the 1.11.2 release, and a teaser for the upcoming
+Puppet 4 support!
+
+#### [Community Demo](https://youtu.be/Z2xKULg5MCA) on 9th June 2016
+
+On this episode of the Foreman Community Demo, we had a look at upcoming tools
+for flexible data exports, a new admin plugin for Hammer, some email styling
+help, new collections for Settings, and a new lock-in feature for Discovery.
+Oh, and 1.12.0 RC1 :)
+
+#### [Foreman Deep Dives: SQL Debugging for Beginners](https://www.youtu.be/kbslq-4EuyY)
+
+Chris Duryee (@beav) joined us for this Deep Dive, in which he gave us an
+in-depth look into SQL debugging. If you've ever wanted to know why certain
+pages are slow to load, this is one to watch!
+
+#### [Case Study on Discovery and Custom UIs with MyHeritage](https://youtu.be/p4-i4y4co0E)
+
+For this case study, we talked with Matt Nicholson (@sjoeboo, author of the
+smart-proxy-{dhcp,dns}-infoblox plugins) about his use of Foreman at Harvard
+University Medical School, how he went about implementing the Infoblox plugins,
+and wider thoughts on extending Foreman in general.
+
+#### [Satellite 6.2 Beta 2 released](https://access.redhat.com/blogs/1169563/posts/2377441)
+
+For those following the downstream releases as well as upstream ones, Satellite
+6.2 Beta 2 is now available for download.
+
+### Upcoming events
+
+#### [Foreman San Francisco Meetup](https://plus.google.com/b/102496134326414788199/events/cngcahgf6f460tnchip0bjb9rh4)
+
+Local meetup for folks in the San Francisco area, along with those visiting for
+DevNation or Red Hat Summit. Come along and have a chat with other Foreman
+users and some of the devs!
+
+#### [Community Demo](https://plus.google.com/events/cupjll3uo0h26aqk89pa4bga5p0) on 7th July 2016
+
+The next Foreman Community Demo will be on Thursday. As ever, you can watch
```

Comment:
«will be on Thursday» maybe Thursday is nos enough? e.g the first Thursday of July?